pkg/apis/storage/types.go

```diff
@@ -511,6 +526,9 @@ type CSINodeSpec struct {
 	// If all drivers in the list are uninstalled, this can become empty.
 	// +patchMergeKey=name
 	// +patchStrategy=merge
+	//
+	// This field is immutable.
```

Comment:
this field is mutable but the item in the list is immutable. 

we can add or delete a item.